index.html

```diff
@@ -59,14 +71,18 @@ <h3 class="contact-title">LET’S CONNECT</h3>
                 </li>
               </ul>
 
-        </section>
+          </section>
+      </div>
+        <div class="all_middle_container">
 
-        <section>
           <div class="card">
-               <figure>
+            <figure class="mobile_project">
               <img src="images/big_icon/tonic.png" alt=" " />
```

Comment:
- [ ] Good job so far, but when you designed it, you missed some issues in the desktop version,  the work section not aligning correctly, and then the about my self section was not intended as the template you chose. Kindly fix :wrench:  the issue addressed in your design as much as possible and stick to your chosen design template.

***Your design***
![Screenshot from 2022-11-11 19-02-09](https://user-images.githubusercontent.com/65129228/201381237-da9b78bc-d032-450c-8142-f0fb9003bdff.png)

- [ ] Kindly use the CSS alignment technologies `grid` and `flex`. To align the projects use `display: grid` for the individual project use `display: flex`

Reply:
![image](https://user-images.githubusercontent.com/4492335/201453697-5b16a6b1-b10b-485d-a284-3a7db00ad683.png)